cfgov/v1/migrations/0006_contactsnippet_contactmolecules.py

```diff
@@ -0,0 +1,101 @@
+                        )
+                    ])
+                ),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='demopage',
+            name='molecules',
+            field=wagtail.wagtailcore.fields.StreamField(
+                [
+                    (b'half_width_link_blob', wagtail.wagtailcore.blocks.StructBlock(
+                        [
+                            (b'heading', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=True)),
+                            (b'content', wagtail.wagtailcore.blocks.RichTextBlock(blank=True)),
+                            (b'links', wagtail.wagtailcore.blocks.ListBlock(wagtail.wagtailcore.blocks.StructBlock(
+                                [
+                                    (b'text', wagtail.wagtailcore.blocks.CharBlock(required=False)),
+                                    (b'url', wagtail.wagtailcore.blocks.URLBlock(required=False))
+                                ], required=False, icon=b'user')))
+                        ])
+                    ),
+                    (b'text_introduction', wagtail.wagtailcore.blocks.StructBlock(
+                        [
+                            (b'heading', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=True)),
+                            (b'intro', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=True)),
+                            (b'body', wagtail.wagtailcore.blocks.RichTextBlock(required=False)),
+                            (b'link_url', wagtail.wagtailcore.blocks.URLBlock(required=False)),
+                            (b'link_text', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=False)),
+                            (b'has_rule', wagtail.wagtailcore.blocks.BooleanBlock(required=False))
+                        ])
+                    ),
+                    (b'image_text_5050', wagtail.wagtailcore.blocks.StructBlock(
+                        [
+                            (b'title', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=True)),
+                            (b'description', wagtail.wagtailcore.blocks.RichTextBlock(blank=True)),
+                            (b'image', wagtail.wagtailimages.blocks.ImageChooserBlock(required=False)),
+                            (b'image_path', wagtail.wagtailcore.blocks.CharBlock(required=False)),
+                            (b'image_alt', wagtail.wagtailcore.blocks.CharBlock(required=False)),
+                            (b'is_widescreen', wagtail.wagtailcore.blocks.BooleanBlock(required=False)),
+                            (b'is_button', wagtail.wagtailcore.blocks.BooleanBlock(required=False)),
+                            (b'link_url', wagtail.wagtailcore.blocks.URLBlock(required=False)),
+                            (b'link_text', wagtail.wagtailcore.blocks.CharBlock(max_length=100, required=False))
+                        ])
+                    )
+                ], blank=True),
```

Comment:
Should probably put `blank=True),` on a new line to make it clear all wrappers have closed